client/media/CVideoHandler.h

```diff
@@ -77,17 +77,20 @@ class CVideoInstance final : public IVideoInstance, public FFMpegStream
 	SDL_Surface * surface = nullptr;
 	Point dimensions;
 
-	/// video playback current progress, in seconds
-	double frameTime = 0.0;
+	/// video playback start time point
+	std::chrono::high_resolution_clock::time_point startTime;
```

Comment:
steady_clock might be more suitable.
We don't need max resolution, but we don't want clock to suddently return smaller time stamp than before